### PR TITLE
Added ssu-align 0.1.1

### PR DESCRIPTION
**Pull request by mworkentine**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/ssu-align/post-link.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+echo "
+ssu-align has been installed.  To run it properly please set the
+environment variable SSUALIGNDIR using one the of the following
```
Comment (bgruening) [inline recipes/ssu-align/post-link.sh]: @mworkentine why is this needed? Can this program not simply use the pwd from the the binary?

Comment (ar0ch) [reply recipes/ssu-align/post-link.sh]: SSUAlign is weird and uses that ENVVAR to find libraries instead of the binary path since the install lets you install them to different places if you want

**Comment by mworkentine** [comment]
Yes, it’s not really ideal at all. I wasn’t sure how to deal with this. Do you know if there are other bioconda packages that have dealt with similar situations?  In the mean time I’ll have a closer look at the source to see if it might be patched.

File: recipes/ssu-align/post-link.sh (hunk 1 above)
Comment (bgruening) [reply recipes/ssu-align/post-link.sh]: Can we include a very tiny bash wrapper for both binaries that do the export and forward all parameters to the original binary. So something like:

* rename original binary to PREFIX/bin/_ssu-align
* create a wrapper called PREFIX/bin/ssu-align that calles _ssu-align but exports the ENV before

**Comment by ar0ch** [comment]
Alternatively you can patch the perl scripts:

```
my $ssualigndir = $ENV{'SSUALIGNDIR'};
```

```
my $ssualigndir = $PREFIX/share/ssu-align-${version};
```

**Comment by bgruening** [comment]
@ar0ch PREFIX might not be available in the runtime env.

**Comment by ar0ch** [comment]
@bgruening Should be available to build.sh for patching though

**Comment by bgruening** [comment]
@ar0ch yes, during build time yes. But then you end up with a path in this file, which is wrong during runtime, right?

**Comment by ar0ch** [comment]
If its patched during build (either a diff just sed the whole line), that path will be correct during runtime

**Comment by mworkentine** [comment]
@bgruening I went with your suggestion for wrapper scripts which are created on the fly in build.sh.
